packages/core/src/compatibility.ts

```diff
@@ -53,7 +53,7 @@ export function handleCompatibility(header: unknown, payload: unknown): UcanPart
 
   // parse either the "ucv" or "uav" as a version in the header
   // we translate 'uav: 1.0.0' into 'ucv: 0.3.0'
-  let version: "0.8.1" | "0.3.0" = "0.8.1"
+  let version: "0.9.1" | "0.3.0" = "0.9.1"
```

Comment:
@blaine Not entirely sure what this does, does it set the default version to 0.9.1?